components/homme/src/theta-l_kokkos/cxx/CaarFunctorImpl.hpp

```diff
@@ -39,7 +39,7 @@ struct CaarFunctorImpl {
 
   struct Buffers {
     static constexpr int num_3d_scalar_mid_buf = 10;
-    static constexpr int num_3d_vector_mid_buf =  3;
+    static constexpr int num_3d_vector_mid_buf =  5;
```

Comment:
Did you check that you can't reuse existing buffers (possible if some variables are in independent regions of code)?

Reply:
Yeah, I could eliminate one by splitting up the parallel_for that mgrad is computed in, but that doesn't seem worth it.

Reply:
Ok, that's fine. Just wanted to be sure you had considered possibilities.